airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/util/HostPortResolver.java

```diff
@@ -12,14 +12,11 @@
 public class HostPortResolver {
 
   public static String resolveHost(GenericContainer container) {
-    return System.getProperty("os.name").toLowerCase().startsWith("mac")
-        ? getIpAddress(container)
-        : container.getHost();
+    return getIpAddress(container);
```

Comment:
This is going to break the clickhouse integration tests on Mac I think.
The original change was #14701 back in July last year

Reply:
source-clickhouse that is